environments/production/app-processes.yml

```diff
@@ -97,6 +99,16 @@ celery_processes:
       pooling: gevent
       concurrency: 100
       num_workers: 2
+  celery6:
+    celery:
+      concurrency: 4
+    case_import_queue:
+      concurrency: 4
+    case_rule_queue:
+      concurrency: 4
+    async_restore_queue:
+      num_workers: 3
```

Comment:
Is there any particular reason to not make this ```concurrency: 6``` with default ```num_workers: 1```?  It seemed from the #celery conversation that this configuration would result in fewer ```MainProcess``` instances and therefore less memory overhead.  Or do we think having more ```MainProcess``` instances gives us some extra form of redundancy?

Reply:
You're right, there's no particular reason I chose to do it this way—I was just copying from the existing definition. I'll go with your suggestion